website/docs/table-group/data-table/data-table-code.md

```diff
@@ -252,81 +278,80 @@ To apply properties to multiple table cells, define `<DataTable.Cell />` with th
 
 :::
 
-## Adding additional elements to table body
+### Custom footer cells
```

Comment:
this seems like it can be renamed to "Custom footer row" and moved to the "Rows" section, because it's about a whole row in reality

Reply:
Hmm, not sure about this. Wiil discuss it with Ilia a little bit.

Reply:
This example are more complicated, hehe. Here is Ilia's comment on how to name this:

> view after main table body which has  css variables for correct width for cell in flex row :smiley:

😁